packages/api-client/src/components/ImportCollection/ImportCollectionModal.vue

```diff
@@ -129,14 +130,45 @@ onUnmounted(() => {
   document.body.classList.remove('has-import-url')
   document.body.classList.remove('has-no-import-url')
 })
+const themeStyleTag = computed(
+  () =>
+    activeWorkspace.value &&
+    shouldShowIntegrationIcon.value &&
+    props.integration &&
+    `<style>${getThemeStyles(props.integration as IntegrationThemeId)}</style>`,
```

Comment:
this is how we inject themes?

Reply:
Yeah... IIRC this is because of some SSR contraints